app.rb

```diff
@@ -9,11 +9,24 @@
 
 get '/start' do
   params[:line]
-  @stations = metro[params[:line].to_sym]
+  @starting_station = metro[params[:line].to_sym]
   erb :start
 end
 
+
 get '/end' do
-  @stations = metro[params[:line].to_sym]
+  @ending_station = metro[params[:line].to_sym]
   erb :end
 end
+
+get '/trip' do
+  @line = metro[params[:line].to_sym]
+
+  # index of starting station
+  @start = params[:start]
+  # index of ending station
+  @end = params[:end]
+
+  # @num_stops =
```

Comment:
subtract the ending index from beginning index, convert to positive if negative.